google-cloud-clients/google-cloud-storage/src/main/java/com/google/cloud/storage/BlobWriteChannel.java

```diff
@@ -83,6 +92,29 @@ public String call() {
     }
   }
 
+  private static String open(final URL signedURL, final StorageOptions options) {
+    try {
+      return runWithRetries(
+          new Callable<String>() {
+            @Override
+            public String call() {
+              String signedURLQuery = signedURL.getQuery();
+              if (!signedURLQuery.startsWith("GoogleAccessId=")
```

Comment:
This wouldn't be compatible with V4 Signing (which will become default in the future), which starts with X-Goog-Algorithm instead of GoogleAccessId. See https://github.com/googleapis/google-cloud-java/pull/4692

Reply:
@JesseLovelace V4 Signing support changes Done